exercises/01_trim_right/main_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "testing"
+
+func TestTrimRight(t *testing.T) {
+	for _, tc := range []struct {
+		name  string
+		input string
+		want  string
+	}{
+		{"empty", "", ""},
+		{"nothing to trim", "abc", "abc"},
```

Comment:
nit: I also recomment to add the following two/three tests:
- `"a"` just a single non-space
- `" "` just a single space
- `"a "` a single non-space + space
Often there are bugs (your implemnetation works just fine though) that work for general cases, but fail for some specific corner-cases. And these are good simple tests to run.